src/main/flight/imu.c

```diff
@@ -166,6 +168,12 @@ void imuInit(void)
     smallAngleCosZ = cos_approx(degreesToRadians(imuRuntimeConfig.small_angle));
     accVelScale = 9.80665f / acc.dev.acc_1G / 10000.0f;
 
+#ifdef USE_GPS
+    canUseGPSHeading = true;
```

Comment:
Always keeping `canUseGPSHeading` is a very bad idea. In fact GPS CoG could only be used when copter is flying forward without sideways drift.